### DISCUSSION TITLE
Self-host server URL does not stick

### DISCUSSION THREAD

**Discussion by jeffshead** [dani-garcia/vaultwarden#4435] [Q&A]
### Subject of the issue
I could have sworn I reported this issue previously, but I cannot find anything under Issues nor Discussions.

For several months, the URL you specify for the self-hosted server does not stick. Once you close the Firefox browser or logout, re-open and try to login, the URL is missing so you have to add it every time you use the Firefox add-on. This affects Android devices, not Windows and it does not affect the Android app, only the Android Firefox browser add-on. I get the same results on every Android device I have tested. I don't know about Macs or iOS.  

### Deployment environment

* vaultwarden version: v1.30.5

* Install method: Docker image

* Clients used:  Android Firefox add-on

* Reverse proxy and version:  NPM 4.11.1

* MySQL/MariaDB or PostgreSQL version:  SQLite

* Other relevant details:

### Steps to reproduce

Setup the Firefox add-on to use a selfhosted server. Add the server's URL. This works just fine until the browser is closed or when you logout. Upon reopening, the server's URL is missing.
### Expected behaviour

Remember server's URL
### Actual behaviour

Server's URL is missing. Doesn't stick after browser's closed.

**Comment by BlackDex** [comment]
If, then nothing we can do about it, is a client issue. And clients are not maintained by this project.
